doc/man/hirte-agent.conf.5.md

```diff
@@ -20,8 +20,7 @@ by `hirte-agent`.
 
 #### **NodeName** (string)
 
-The unique name of this agent. The option doesn't have a default value, it's mandatory to set this option for each
-`hirte-agent`.
+The unique name of this agent. The option defaults to the system's hostname.
```

Comment:
Could you please update description also in the following files?

https://github.com/containers/hirte/blob/main/config/agent/agent.conf#L9
https://github.com/containers/hirte/blob/main/config/agent/hirte-default.conf#L5
https://github.com/containers/hirte/blob/main/doc/man/hirte-agent.1.md?plain=1#L40

Reply:
Aren't those generated some how.  I updated the two spots I see NodeName defined.

Reply:
The blob subdir is not in the github checkout?

Reply:
blob subdir is only used when fetching current files from GH, those files are included in the project:

```shell
$ git grep NodeName
.github/workflows/e2e.yml:               bash -c   "sed -i \"s/\(AllowedNodeNames=\)\(.*\)/\1foo,bar/\" /etc/hirte/hirte.conf"
.github/workflows/e2e.yml:               bash -c "sed -i \"s/\(NodeName=\)\(.*\)/\1bar/\" /etc/hirte/agent.conf"
.github/workflows/e2e.yml:               bash -c "sed -i \"s/\(NodeName=\)\(.*\)/\1foo/\" /etc/hirte/agent.conf"
config/agent/agent.conf:NodeName=
config/agent/hirte-default.conf:NodeName=
config/hirte/hirte-default.conf:# hirte manager. These names are defined in the agent's configuration file under `NodeName` option.
config/hirte/hirte-default.conf:AllowedNodeNames=
config/hirte/hirte.conf:# manager. These names are defined in the agent's configuration file under the `NodeName` option.
config/hirte/hirte.conf:AllowedNodeNames=
doc/man/hirte-agent.conf.5.md:#### **NodeName** (string)
doc/man/hirte-agent.conf.5.md:NodeName=agent-007
doc/man/hirte-agent.conf.5.md:NodeName=agent-007
doc/man/hirte.conf.5.md:#### **AllowedNodeNames** (string)
doc/man/hirte.conf.5.md:in the list can connect to `hirte` manager. These names are defined in the agent's configuration file under `NodeName`
doc/man/hirte.conf.5.md:AllowedNodeNames=agent-007,agent-123
src/libhirte/common/cfg.h:#define CFG_NODE_NAME "NodeName"
src/libhirte/common/cfg.h:#define CFG_ALLOWED_NODE_NAMES "AllowedNodeNames"
```

Reply:
Right and I think I fixed all of those files.

Reply:
hm, I don't see the file `config/agent/agent.conf` changed in this PR, did you forget to push it? :)

Reply:
Updated, what is the difference between hirte-agent.conf and agent.conf?

Reply:
https://github.com/containers/hirte/issues/147 describes location and purpose of configuration files